grostl/tests/lib.rs

```diff
@@ -0,0 +1,36 @@
+#![no_std]
+#[macro_use]
+extern crate crypto_tests;
+extern crate generic_array;
+extern crate grostl;
+
+use crypto_tests::hash::{Test, main_test};
+use generic_array::typenum::{U28, U32, U48, U64};
```

Comment:
You can use type aliases instead of using `GrostlSmall` and `GrostlBig` directly. In my opinion it will look a bit nicer.